crates/paralegal-policy/src/context.rs

```diff
@@ -300,19 +300,13 @@ impl Context {
         }
 
         // Special case if src is a CallArgument and sink is a CallSite, flows_to is true if they are on the same CallSite.
-        match edge_type {
-            EdgeType::Data | EdgeType::DataAndControl => {
-                if let (
-                    NodeType::CallArgument(DataSink::Argument { function, .. }),
-                    NodeType::CallSite(cs),
-                ) = (src.typ, sink.typ)
-                {
-                    if *function == *cs {
-                        return true;
-                    }
-                }
-            }
-            _ => (),
+        if matches!(
```

Comment:
Because this is a macro call you will have to format this by hand I think. (Just reduce the indentation on the arguments a bit).

Reply:
done